recipes/isitgr/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "isitgr" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mishakb/ISiTGR/archive/{{ version }}.tar.gz
+  sha256: ebf53b46ce3682a0154f3e713e8d320f5f7588fe1cb7a6fe5c9db8d1cad78b59
+
+build:
+  number: 0
+  skip: True  # [py2k or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - git
+    - make
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - numpy
+    - six  # TODO: remove for next update
+    - mock
+    - setuptools  # for an import of pkg_resources
+    - sympy >=1.0
+    - scipy >=1.0
+    - {{ compiler('fortran') }}  # [not win]
+
+test:
+  imports:
+    - isitgr
+  commands:
+    - python {{ RECIPE_DIR }}/test_isitgr.py
+
+about:
+  home: https://github.com/mishakb/ISiTGR
+  license: MIT AND GPL-3.0-or-later
+  license_file:
+    - LICENSE
+    - camb/LICENCE.txt
```

Comment:
This licence can be found in the source tarball so it can be deleted from the recipe.

After that’s done it looks good to merge :+1:

Reply:
Done.